R/pkg/R/sparkR.R

```diff
@@ -155,6 +155,9 @@ sparkR.sparkContext <- function(
 
   existingPort <- Sys.getenv("EXISTING_SPARKR_BACKEND_PORT", "")
   if (existingPort != "") {
+    if(sparkPackages != ""){
+        warning("--packages flag should be used with with spark-submit")
```

Comment:
@shivaram @felixcheung how about something like `sparkPackages cannot be used as an argument within sparkR.init please use the --packages flag while using spark-submit or sparkR shell`